tiledb/sm/query/query.h

```diff
@@ -85,12 +86,12 @@ class Query {
    * @tparam T
    * @param attribute name of buffer to retrieve
    * @return a std::pair which contains two pairs. First pair is a pointer to
-   * the value buffer and its size. The second pair is a pointer to the offset
-   * buffer and its size. Note the first pair (the value buffer) is of type T
-   * where the second pair (the offset buffer) is always type uint64_t
+   * the offset buffer and its size. The second pair is a pointer to the values
```

Comment:
@Shelnutt2 a belated comment (apologies for that): It seems that this function returns the buffer pointers along with the corresponding **number of elements** instead of their sizes in bytes. This is fine, but perhaps we should make it clear in the function doc. Also, throughout the rest of the code, we use buffer sizes always in bytes. Perhaps it is a good idea to be consistent.

Reply:
I agree that consistency is something we should aim for. We need to expose a get buffers via the c_api, when we determine how we want to do that, we can adjust this function accomidate that.